### ISSUE TITLE
Language is broken...??

### ISSUE THREAD

**Issue by CloudeLecaw** [BenWoodworth/FastCraftPlus#35]
I already wrote on spigot about:

http://www2.pic-upload.de/img/30361779/uiuiui.jpg

<details><summary>config.yml</summary>
```
# language          If the language specified does not exist, a new language file will be added to the
#                   lang folder in the FastCraftPlus plugin directory.
#                   Available languages: English (EN), German (DE), Russian (RU), Czech (CS)
#
# command-compat    Do not open the FastCraft+ GUI for these commands.

language: DE
command-compat:
- /recipe
- /erecipe
- /formula
- /eformula
- /method
- /emethod
- /recipes
- /erecipes
- /furniture recipe
```
</details>

You see my current config file.
I looked into the jarfile if a DE.yml is placed there..
it is.. :3

**Comment by CloudeLecaw** [comment]
i hope u find the reason with the language. most just not may see this because they using it in english

**Comment by BenWoodworth** [comment]
I fixed it for v0.17 :)
